drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/fake_data.rb

```diff
@@ -0,0 +1,219 @@
+module AllNeighborsSystemDashboard
```

Comment:
Are you just leaving this in case we need it for front end development?

Reply:
Leaving until we have the internal dashboard version as well.